wandb/sdk/interface/artifacts.py

```diff
@@ -637,6 +637,66 @@ def wait(self) -> "Artifact":
         """
         raise NotImplementedError
 
+    def __getitem__(self, name: str) -> Optional[WBValue]:
+        """
+        Gets the WBValue object located at the artifact relative `name`.
+
+        NOTE: This will raise an error unless the artifact has been fetched using
```

Comment:
This will also work if we call `artifact.wait()`, right?

Reply:
correct, i can update

Reply:
wait, this seems correct - it is the same as `get()`. What exactly are you suggesting here?

Reply:
Whoops, I should have updated this docstring when I added wait. Could you update this to say:

```
        NOTE: This will raise an error unless the artifact has been fetched using
        `use_artifact`, fetched using the API, or `wait()` has been called.
```